src/messages/rdf.js

```diff
@@ -0,0 +1,41 @@
+import { gettext as _, singleLineString } from 'utils';
+
+
+export const TAG_NOT_ALLOWED_HIDDEN = {
+  code: 'TAG_NOT_ALLOWED_HIDDEN',
+  message: _('The <hidden> tag is not allowed'),
+  description: _(singleLineString`Your RDF file contains the <hidden> tag,
+    which is not allowed in an Add-on.`),
+};
+
+// NOTE: I didn't want to change the commonality of messages without discussion,
```

Comment:
Haha, nevermind, I was thinking in Ruby and trying to metaprogram too much. I'll make them up-front, new commit incoming in a few :smile: